test/compilable/minimal3.d

```diff
@@ -0,0 +1,13 @@
+// DFLAGS:
+// REQUIRED_ARGS: -defaultlib=
```

Comment:
`-betterC`?

Reply:
Not testing betterC, testing empty object module.  Otherwise the second part of the tweaked condition won't be hit.

Reply:
OK